contrib/win32/win32compat/win32-utf8.c

```diff
@@ -65,7 +65,6 @@ snmprintf(char *buf, size_t len, int *written, const char *fmt, ...)
 void
 msetlocale(void)
 {
-	// allow console output of unicode characters
-	SetConsoleOutputCP(CP_UTF8);
+	// Do nothing
```

Comment:
This is a common code. 

Removing this code will have impact scp, sftp, ssh-agent.